docs/examples/preprocess.py

```diff
@@ -0,0 +1,55 @@
+"""Generate GitHub README's from _index.rst files
+GitHub doesn't support include of other files, even of the same type and
+location, so this file generates a README.rst with files content embedded
+"""
+from glob import glob
+from pathlib import Path
+import shutil
+import subprocess
+
+
+def preprocess():
```

Comment:
Wow this is really cool!

One question though. If this gets called to generate the README.rst files automatically, then why are they included in the PR? Shouldn't this only include the _index.rst files?

Reply:
Only because otherwise the README would be visible on github